tests-e2e/ocw-ci-test-course/footer.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { test, expect } from "@playwright/test"
+import { CoursePage } from "../util"
+import { SOCIAL_LINKS } from "../util/constants"
+
+test("Social links exist and open correct social media pages", async ({
+  page
+}) => {
+  const course = new CoursePage(page, "course")
+  await course.goto()
+
+  for (const [name, url] of SOCIAL_LINKS) {
+    const link = page
+      .locator(".footer-main-content")
+      .getByRole("link", { name, exact: true })
+    await expect(link).toBeVisible()
+    await expect(link).toHaveAttribute("href", url)
```

Comment:
Originally, I simulated a click and ran assertions on the navigation (i.e. open tab's URL). However, these external links can have uncontrollable behaviors.

For instance, I can open the [Instagram link](https://www.instagram.com/mitocw/) without being signed in but, in GitHub Action runners, this URL redirects to the login screen.

For that reason, I have not placed an assertion on the `href` attribute. Feel free to suggest improvements.